packages/hash/api/src/db/adapter.ts

```diff
@@ -60,17 +60,33 @@ export type EntityWithOutgoingEntityIds = DbEntity & {
 
 export type DBLink = {
   linkId: string;
+  linkVersionId: string;
   path: string;
   index?: number;
   sourceAccountId: string;
   sourceEntityId: string;
+
   appliedToSourceAt: Date;
-  appliedToSourceBy: string;
+  appliedToSourceByAccountId: string;
+
   removedFromSourceAt?: Date;
-  removedFromSourceBy?: string;
+  removedFromSourceByAccountId?: string;
```

Comment:
Yeah, it also helps distinguish between when we are using an instance of a model class (`createdBy: User`) and when we're just using the identifier (`createdByAccountId: string`).

Do you think I should update the column names in the db to also reflect this? (i.e. `update_by_account_id` instead of `updated_by`)